ui/app/components/auth-config-form/options.js

```diff
@@ -30,12 +30,20 @@ export default AuthConfigComponent.extend({
     waitFor(function* () {
       const data = this.model.config.serialize();
       data.description = this.model.description;
+      data.user_lockout_config = {};
 
       // token_type should not be tuneable for the token auth method.
       if (this.model.methodType === 'token') {
         delete data.token_type;
       }
 
+      this.model.userLockoutConfig.apiParams.forEach((attr) => {
```

Comment:
I debated doing this in the serializer...but these models are already quite tangled up between tune actions, mount config relationships and auth mounts vs secret engine mounts. 

Since there was already some data manipulation here (adding `description`) above this seemed like a reasonable place for this so that moving the logic when we refactor would be easier.

Reply:
Is it worth maybe adding test coverage to make sure that the values are sent in the tune request as expected by triggering a save in the component?

Reply:
yes! I was planning to add those but was waiting for folks to approve this logic happening here instead of the serializer.